js/checklist.js

```diff
@@ -20,7 +25,9 @@
 	 function modify_import_content_page($) {
```

Comment:
Yeah, it probably should live in a separate file since that code isn't really part of the checklist module. Good suggestion.